src/NUnitFramework/framework/Internal/TestProgressReporter.cs

```diff
@@ -175,6 +181,17 @@ public void TestFinished(ITestResult result)
             }
         }
 
+        private static ITest GetTestAssembly(ITest test)
```

Comment:
A bit of code duplication here. Remove this one and then make GetTestAssembly on TestResult internal.